examples/mobile/build.rs

```diff
@@ -0,0 +1,27 @@
+//! A build script to copy libc++_shared.so to the target directory.
+use std::{
+    env,
+    path::{Path, PathBuf},
+};
+
+fn main() {
+    if env::var("CARGO_CFG_TARGET_OS").unwrap() == "android" {
+        android();
+    }
+}
+
+fn android() {
+    println!("cargo:rustc-link-lib=c++_shared");
+
+    if let Ok(output_path) = env::var("CARGO_NDK_OUTPUT_PATH") {
+        let sysroot_libs_path = PathBuf::from(env::var_os("CARGO_NDK_SYSROOT_LIBS_PATH").unwrap());
+        let lib_path = sysroot_libs_path.join("libc++_shared.so");
+        std::fs::copy(
+            lib_path,
+            Path::new(&output_path)
+                .join(env::var("CARGO_NDK_ANDROID_TARGET").unwrap())
+                .join("libc++_shared.so"),
```

Comment:
this is bad practice in build scripts:
https://doc.rust-lang.org/cargo/reference/build-scripts.html#outputs-of-the-build-script
> Scripts should not modify any files outside of that directory.

Reply:
> this is bad practice in build scripts: https://doc.rust-lang.org/cargo/reference/build-scripts.html#outputs-of-the-build-script
> 
> > Scripts should not modify any files outside of that directory.

I copird it directly from cargo-ndk readme🤔

Reply:
do you know why they don't just copy it themselves instead of requiring a build script for that?

Reply:
> do you know why they don't just copy it themselves instead of requiring a build script for that?

I don't know, they didn't say it in it's readme. They just said add this script if you want it

Also the script here is edited by [this pr](https://github.com/Litttlefish/bevy/pull/1)(removed a ref, I don't know what this will do), it was:
>Path::new(&output_path)
.join(&env::var("CARGO_NDK_ANDROID_TARGET").unwrap())
               .join("libc++_shared.so"),

Reply:
I removed it, so this is resolved I think